src/main/fc/rc_adjustments.h

```diff
@@ -58,6 +58,9 @@ typedef enum {
     ADJUSTMENT_PITCH_F,
     ADJUSTMENT_ROLL_F,
     ADJUSTMENT_YAW_F,
+#ifdef USE_OSD_PROFILES
```

Comment:
Same as above, the enumeration cannot be conditionally compiled. The value will be necessary as it's referenced from the `defaultAdjustmentConfigs` entry.

Reply:
Done